sdk/src/sdk/middleware/DaprMiddleware.cpp

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2022-2023 Robert Bosch GmbH
```

Comment:
It is the new created file. Shall be only 2023

Reply:
I put 2022 here also as I moved the `waitForSidecar` here, which is from 2022